src/appleseed-max-impl/appleseedoslplugin/oslparamdlg.cpp

```diff
@@ -50,20 +50,20 @@ OSLParamDlg::OSLParamDlg(
     IMtlParams*             imp, 
     MtlBase*                map,
     const OSLShaderInfo*    shader_info)
-    : m_hmedit(hwMtlEdit)
-    , m_bump_pmap(nullptr)
-    , m_pmap(nullptr)
-    , m_imp(imp)
-    , m_osl_plugin(map)
-    , m_shader_info(shader_info)
-    , m_col_1(10)
-    , m_col_2(85)
-    , m_col_3(122)
-    , m_col_4(159)
-    , m_label_width(76)
-    , m_edit_width(25)
-    , m_edit_height(10)
-    , m_tex_width(84)
+  : m_hmedit(hwMtlEdit)
+  , m_bump_pmap(nullptr)
+  , m_pmap(nullptr)
+  , m_imp(imp)
+  , m_osl_plugin(map)
+  , m_shader_info(shader_info)
+  , m_col_1(10)
```

Comment:
Are these variables or constants? If they are constants, maybe they can be removed from the class and be declared as constants where they're used?